TESTS/host_tests/pyusb_basic.py

```diff
@@ -1369,8 +1369,7 @@ def ep_test_abort(dev, log, verbose=False):
             payload_size = (NUM_PACKETS_UNTIL_ABORT + NUM_PACKETS_AFTER_ABORT) * ep_out.wMaxPacketSize
             num_bytes_written = 0
             while num_bytes_written < payload_size:
-                payload_out = array.array('B', (num_bytes_written/ep_out.wMaxPacketSize
-                                                for _ in range(ep_out.wMaxPacketSize)))
+                payload_out = array.array('B', ([num_bytes_written//ep_out.wMaxPacketSize] * ep_out.wMaxPacketSize))
```

Comment:
I would prefer to have a specific PR for this "common" test part ?
(to be tested with all other targets?)